lib/blueprint-compiler.yaml

```diff
@@ -0,0 +1,12 @@
+name: blueprint-compiler
+buildsystem: meson
+cleanup:
+  - '*'
+sources:
+  - type: git
+    url: https://gitlab.gnome.org/jwestman/blueprint-compiler
+    tag: v0.16.0
+    x-checker-data:
+      type: git
+      tag-pattern: ^v([\d.]+)$
```

Comment:
does this really need to be updated? Given that it is a build dependency and that it might actually break.

Reply:
Unpatched build dependencies can also get you pwned. À la  xz utils backdoor...
If a PR with a braking change is ever raised by the checker, then that obviusly won't be merged until upstream adapts.

Reply:
Much like a pwned one can be automatically grabbed by this unecessary update. That would have accelerated the xz backdoor.

Reply:
Nothing is grabbed automatically.
The notification and the pr is automatic. The decision to test, accept and merge is manual.